nixos/services/percona.nix

```diff
@@ -50,34 +42,15 @@ let
     chmod 0755 /run/mysqld
     chown -R ${cfg.user} /run/mysqld
 
-    # Generate mysql root password file if it's not present.
-    if [[ ! -f ${cfg.rootPasswordFile} ]]; then
-      pw=`${pkgs.apg}/bin/apg -a 1 -M lnc -n 1 -m 12`
-      echo -n "''${pw}" > ${cfg.rootPasswordFile}
-    fi
-    chown root:service ${cfg.rootPasswordFile}
-    chmod 660 ${cfg.rootPasswordFile}
-
-    pw=$(<${cfg.rootPasswordFile})
-    cat > /root/.my.cnf <<__EOT__
+    cat > /srv/mysql/.my.cnf <<__EOT__
```

Comment:
consider templating `cfg.user` and their homedir here to make this more consistent. This also makes it more clear that we are relying on homedir config file discovery logic.